### DISCUSSION TITLE
How to export the onnx model from detectron2 and use it in the c++ version of onnxruntime

### DISCUSSION THREAD

**Discussion by Myxrf** [facebookresearch/detectron2#3543] [General]
How to export the onnx model from detectron2 and use it in the c++ version of onnxruntime，
The model officially exported by detectron2 seems to be unable to be used on onnxruntime. Is there a way?

**Comment by thiagocrepaldi** [comment]
I am working on adding support to ONNX export (with and without caffe2) https://github.com/facebookresearch/detectron2/pull/4120 and https://github.com/facebookresearch/detectron2/pull/4153/ to detectron, so that it can run on ONNX Runtime. 

Hopefully it gets accepted soon
